src/SupportUsBundle/Resources/views/Default/individual.html.twig

```diff
@@ -25,7 +25,7 @@
     <button type="button" class="btn btn-support-us btn-individual-voter">
         <span class="plan-title">Voter</span>
         <hr>
-        <span>30 € / year</span>
+        <span>30 €</span>
```

Comment:
Why only on voter ?

Reply:
\<Voter: membership is per year
 \>Voter: support (name, logo...) is per year.
Only vote is "unlimited".

Reply:
But none are subscriptions and voter include membership, so I think it would be better to have per year for all or nothing for all

Reply:
changed to nothing for all.